storage/src/context_action_storage.rs

```diff
@@ -72,6 +73,8 @@ pub struct ContextActionStorage {
     context_by_type_index: ContextActionByTypeIndex,
     kv: Arc<ContextActionStorageKV>,
     generator: Arc<SequenceGenerator>,
+    last_block_hash: Arc<Option<BlockHash>>,
+    next_block_action_id: Arc<AtomicU64>,
```

Comment:
@adonagy 
why do you need `next_block_action_id`? I dont see any usage of this attribute

Reply:
There is indeed usage, please look at that `put_action` function bellow. It is used to track the actions of one block and give them the `block_action_id`